app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less

```diff
@@ -53,6 +53,17 @@
         }
     }
 
+    .products-grid.wishlist {
+        .product-item {
```

Comment:
Can these styles be moved to line 128 (.products-grid.wishlist { .product { &-item)?

Reply:
I have applied changes as per your suggestion